users/maxitect/styles.css

```diff
@@ -0,0 +1,33 @@
+h1, h2, p {
+  font-family: Necto Mono;
+}
+
+h1 {
+  margin: 1em auto;
+  text-align: center;
+  width: 60vw;
+}
+
+.central {
+  text-align: center;
+}
+
+.flex-container {
+  padding: 0;
```

Comment:
It could be worth a general CSS reset so that you don't have to do this for each class

There's a good one at: https://meyerweb.com/eric/tools/css/reset/

Reply:
Thanks Jack, indeed there is one property and value set that is repeated in 2 different classes, I will take a look at refactoring it to be a bit more efficient. Is it necessary to reset all the built-in CSS classes even if I am not using them yet?